src/main/java/com/mitchej123/hodgepodge/mixins/early/minecraft/fastload/MixinEntityPlayerMP.java

```diff
@@ -62,7 +62,7 @@ public MixinEntityPlayerMP(World world, GameProfile profile) {
      * mixins that target the tail of this function, but other solutions are similarly invasive.
      */
     @Inject(
-            method = "onUpdate",
```

Comment:
Please add a brief comment on what localOnUpdate is, because it's not obvious from the mixin'd class